modules/serverless-creating-broker.adoc

```diff
@@ -5,30 +5,12 @@
 [id="serverless-creating-broker_{context}"]
 = Creating a broker manually
 
-To create a broker, you must create a `ServiceAccount` for each namespace and give that `ServiceAccount` the required RBAC permissions.
+The following sections describe how to create a broker.
 
 .Prerequisites
-* Knative Eventing installed, which includes the `ClusterRole`.
+* Knative Eventing installed.
 
 .Procedure
-. Create the `ServiceAccount` objects:
-+
-----
-$ oc -n default create serviceaccount eventing-broker-ingress
-$ oc -n default create serviceaccount eventing-broker-filter
-----
-
-. Give those objects RBAC permissions:
-+
-----
-$ oc -n default create rolebinding eventing-broker-ingress \
-  --clusterrole=eventing-broker-ingress \
-  --serviceaccount=default:eventing-broker-ingress
-$ oc -n default create rolebinding eventing-broker-filter \
-  --clusterrole=eventing-broker-filter \
-  --serviceaccount=default:eventing-broker-filter
-----
-
 . Create the broker:
 +
 ----
```

Comment:
CLI commands need to start with $ if you do not need root access or # if you do.

Reply:
@kalexand-rh I did not add this, so I do not know on what "style guide" that was created.

this PR mainly removes tech instructions, that are not needed for our 1.8.0 openshift serverless release